config/whitelist.json

```diff
@@ -1005,6 +1005,15 @@
               "0x7bf2d6d4": "swapMulti((address,uint256,address)[],(address,uint256,address)[],uint256,bytes,address,uint32)",
               "0x84a7f3dd": "swapMultiCompact()"
             }
+          },
+          {
+            "address": "0x0d05a7d3448512b78fa8a9e46c4872c88c4a0d05",
+            "functions": {
+              "0x83bd37f9": "swapCompact()",
+              "0x84a7f3dd": "swapMultiCompact()",
+              "0x30f80b4c": "",
```

Comment:
Hey @melianessa 

why do some of these function selectors not have function names?
4bytes is able to resolve this one for example. Could you please check if your script works correctly?

<img width="2912" height="1172" alt="image" src="https://github.com/user-attachments/assets/5cbb9d53-0270-4174-b93e-fe22769af7ab" />

<img width="1750" height="624" alt="image" src="https://github.com/user-attachments/assets/fef5a6c5-22b2-4db8-9690-6383a69bf3ad" />